tests/flashtest.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Library         SSHLibrary
+
+*** Test Cases ***
+Open Connection and Log In
+    Open Connection  %{OPI_IP}    timeout=360 seconds
+    Login  root    armbian1234
+    
+Flash firmware
+	# turn APU on and put it in S5 state
+	Write    ./relays.sh ON\n
+	# wait for diodes to turn off
```

Comment:
It would need additional script/app to check their state, most certainly. We can leave it this way for now, but let's have in mind, we have such possibility, if the timeout will be varying wildly between the platforms.